README.md

```diff
@@ -1,88 +1,89 @@
-# CAR Mirror over HTTP v0.1.0
+# CAR Mirror over HTTP v0.2.0
 
 ## Authors
 
 * [Brooklyn Zelenka], [Fission]
 
 ## Dependencies
 
+- [CAR Mirror]
 - [CARv1]
 - [CBOR]
 - [HTTP/2]
 - [IPLD Schema]
 
 ## Language
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119].
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14] when, and only when, they appear in all capitals, as shown here.
 
-# 0. Abstract
+# 0 Abstract
 
-CAR Mirror is transport agnostic, and MAY be implemented with a RESTful HTTP client/server. This specification describes the protocol for CAR Mirror over HTTP.
+This specification describes the protocol for synchronous [CAR Mirror] over [HTTP/2].
 
-# 1. Pull
+# 1 Introduction
 
-## 1.1 Endpoint
+[CAR Mirror] describes a generic method extending CAR files to efficiently mirror some IPLD graph state on a peer machine. It is transport agnostic.
+
+HTTP is a very mature protocol that connects the applciations and devices that developers — especially web developers — use most fequently. This spec defines how to perform multi-round, syncronous CAR Mirror over HTTP/2.
+
+# 2. Pull
+
+## 2.1 Endpoint
 
 The endpoint MAY be placed at any route, but it is RECOMMENDED to be exposed at:
 
 ```http
-POST /api/v0/dag/pull?stream=${bool}
+POST /api/v0/dag/pull
 ```
 
-Note the absence of a field for the requested URL. Since multiple CID roots MAY be requested at once, this information is instead located in the Requestor Payload.
+Since multiple CID roots MAY be requested at once, this information is instead located in the Client Payload.
 
-#### 1.1.1 `stream` Parameter
+## 2.2 Client Pull Request
 
-This field specifies if the Provider should use discrete or streaming. `stream` MUST default to `false`. If streaming is set to `true`, the response MUST be a streaming CAR file and transmitted over [HTTP Streaming]. Both the Requestor and Provider MUST use [HTTP/2].
-
-## 1.2 Requestor Payload
-
-The requestor payload MUST be serialized as [CBOR]. 
+The request MUST be serialized as [CBOR]. 
 
 ```ipldsch
 type PullRequest struct {
-  rs [Link]  -- Requested CID roots
+  rs [&Any]  -- Requested CID roots
```

Comment:
I think `Link` is [valid IPLD schema](https://ipld.io/docs/schemas/features/links/), and I'd personally prefer it over `&Any`, as I think it's clearer.

Reply:
I love how everyone has a different opinion on this :P Irakli just made the opposite comment on one of the UCAN specs. I just want to pick one as a matter of style. To Irakli's point, `&Any` is more consistent / you don't have to understand more concepts versus other kinds of links.

Reply:
Is "`Link` vs `&Any`" the new "tabs vs spaces"? 🤔